### DISCUSSION TITLE
Pypi Cache: The value of "length" is out of range.

### DISCUSSION THREAD

**Discussion by SimFG** [community/community#59393] [Actions] [answered]
### Select Topic Area

Bug

### Body

An error occurs when using the cache, it is normal if the cache is not used

![image](https://github.com/community/community/assets/21985684/96d3fb86-a386-4473-a9dc-6be51de90690)

**Comment by OptimusPrinceps** [comment, marked as answer]
You are using `setup-python@v2`, try updating to `v4`

**Reply by SimFG** [reply]
Thanks you!!!
